examples/volumes/glusterfs/README.md

```diff
@@ -26,9 +28,9 @@ Here is a snippet of [glusterfs-endpoints.json](glusterfs-endpoints.json),
 
 ```
 
-The "IP" field should be filled with the address of a node in the Glusterfs server cluster. In this example, it is fine to give any valid value (from 1 to 65535) to the "port" field.
+The `IP` field should be filled with the addresses of the nodes in the GlusterFS server cluster. It is fine to provide any valid value (from 1 to 65535) in the `port` field.
```

Comment:
Done. Expanded the snippet from the full json file to show the complete `subsets` part.